Visma.net/lib/ShipmentActionResult.cs

```diff
@@ -0,0 +1,14 @@
+using Newtonsoft.Json;
+using System;
+
+namespace ONIT.VismaNetApi.Lib
+{
+    public class CreateShipmentActionResult : VismaActionResult
+    {
+        [JsonProperty("referenceNumber")]
+        public string ReferenceNumber { get; internal set; }
```

Comment:
```suggestion
        public string referenceNumber { get; internal set; }
```

Reply:
Why this should be named different way than other properties in code?